src/core_plugins/console/api_server/es_6_0/ingest.js

```diff
@@ -344,6 +420,10 @@ const pipelineDefinition = {
   version: 123,
 };
 
+const commonPipelineParams = {
```

Comment:
This needs to be defined above where it is used (just move this to the top).  Also, there needs to be a semicolon after the `}` closing this object.

Reply:
Fixed in https://github.com/elastic/kibana/pull/24100/commits/11c0ac89f54ec3cde45251d1c4686d720dc9593f